rfcs/20181016-replicator.md

```diff
@@ -0,0 +1,1014 @@
+# Distribution Strategy - Revised API
+
+| Status        | Proposed                                                                  |
```

Comment:
@guptapriya please amend to "Accepted" and then we can merge this, thank you!

Reply:
Great, just did that. Thanks @ewilderj !